cmd/reconciler/start/service/integration_test.go

```diff
@@ -131,6 +131,7 @@ func post(t *testing.T, testCase testCase) interface{} {
 	resp, err := http.Post(urlReconcilerRun, "application/json",
 		bytes.NewBuffer(jsonPayload))
 	require.NoError(t, err)
+	defer resp.Body.Close()
```

Comment:
We need to also check `http.Get()` and close the response body.

Reply:
Nice hint! Will also close them :D